### DISCUSSION TITLE
Do I still need 'terraform fmt' if I use tflint?

### DISCUSSION THREAD

**Discussion by mwgamble** [terraform-linters/tflint#2010] [Q&A]
Does tflint fully replae what I get from running `terraform fmt`? Can I just use tflint and not worry about `terraform fmt`?

**Comment by wata727** [comment]
Generally speaking, TFLint is not a replacement for `terraform fmt`. At least, the officially provided rulesets cannot fully cover the functionality. In theory it is possible to develop such a ruleset, but not to my knowledge.

For reference, the company I work for runs both TFLint and `terraform fmt` in a CI pipeline.
